packages/opentelemetry-core/test/correlation-context/HttpCorrelationContext.test.ts

```diff
@@ -164,6 +164,7 @@ describe('HttpCorrelationContext', () => {
       const extractedSpanContext = getCorrelationContext(
         httpTraceContext.extract(Context.ROOT_CONTEXT, carrier, defaultGetter)
       );
+      console.log(extractedSpanContext);
```

Comment:
If this is not a necessary print, can we remove it?

Reply:
Not necesary hahaha sorry.

Reply:
Done